builders/interact/ethereum-api/dev-env/remix.md

```diff
@@ -0,0 +1,124 @@
+---
+title: Deploy Smart Contracts with Remix
+description: Learn how to use one of the most popular Ethereum developer tools, the Remix IDE, to interact with your Tanssi EVM ContainerChain.
+---
+
+# Using Remix to Deploy to your Tanssi EVM ContainerChain
+
+## Introduction {: #introduction } 
+
+Developers building dApps on top of Tanssi EVM ContainerChains can use [Remix](https://remix.ethereum.org/){target=_blank}, one of the most popular Ethereum development environments, to build, compile, and deploy their smart contracts. Remix can be used with any EVM ContainerChain, thanks to the seamless compatibility of Tanssi EVM ContainerChains. 
+
+This guide walks through the process of creating and deploying a Solidity-based smart contract to the Tanssi Dancebox TestNet using the Remix IDE. This guide can be adapted for your own Tanssi EVM ContainerChain by simply adding the RPC URL of your ContainerChain to your EVM Wallet and switching networks to it.  
+
+## Checking Prerequisites {: #checking-prerequisites } 
+
+For the purposes of this guide, you'll need to have MetaMask installed and configured to work with the Tanssi Dancebox TestNet. You can follow [this guide to configure MetaMask for Tanssi](/builders/interact/ethereum-api/wallets/metamask/){target=_blank}.
```

Comment:
This should be configured to connect to their container chain instead of the dancebox testnet, right?